addon/formatters/-base.js

```diff
@@ -65,7 +65,7 @@ const FormatBase = Ember.Object.extend({
 });
 
 FormatBase.reopenClass({
```

Comment:
The only issue is it's going to be used on things that aren't time-based (ie, format-message).  And also not sure how we need to handle when time is used as part of a ICU string.

ie, 'order by {deadline, date, time}'